### ISSUE TITLE
lip_loss too large

### ISSUE THREAD

**Issue by luqianX** [Sxjdwang/TalkLip#33]
why my lip_loss is >90 even for gt videos?
should not LabelSmoothedCrossEntropyCriterion in fairseq be near -log(1/n)?

**Comment by Sxjdwang** [comment]
The loss you describe is normal. In my training, the lip loss on the validation set is still larger than 200 after training with a batch size of 8.
